examples/genes_analysis.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Analyzing genes of interest with SolveBio"
+output: rmarkdown::html_document
+---
+```{r, message = FALSE, warning=FALSE}
+rm(list=ls())
+library(solvebio)
+library(data.table)
+```
+login to SolveBio:
+```{r eval=FALSE}
+login(api_key="<api_key>", api_host="<api_host>")
+```
+Define a list of genes for the analysis and create a filter that will be used to query SolveBio datasets:
+```{r}
+# random sample from clinvar gene_symbols generated by running: genes = sample(unlist(combined_dt[,gene_symbol]),15)
+genes = c("MYBPC3", "RDH5", "MYO15A", "FZD4", "DMD", "GLIS3", "ISPD", "C3", "APOB", "ATP6V0A2", "KIF5A", "SNCA", "LDLR", "ANK1", "FBN2")
+genes_filter  = '[["gene_symbol__in",["MYBPC3", "RDH5", "MYO15A", "FZD4", "DMD", "GLIS3", "ISPD", "C3", "APOB", "ATP6V0A2", "KIF5A", "SNCA", "LDLR", "ANK1", "FBN2"]]]'
```

Comment:
is there a way to generate the list from the `genes` variable instead of duplicating it?